dataprofiler/profilers/float_column_profile.py

```diff
@@ -28,7 +30,26 @@ def __init__(self, name, options=None):
                              " FloatOptions.")
         NumericStatsMixin.__init__(self, options)
         BaseColumnPrimitiveTypeProfiler.__init__(self, name)
-        self.precision = 0
+
+        self.precision = {
+            'min': None,
+            'max': None,
+            'mean': None,
+            'var': None,
+            'std': None,
+            'sum': None,
+            'sample_size': None,
+            'margin_of_error': None,
+            'confidence_level': 0.999
```

Comment:
usually, the confidence level is 95% or 99%. Can we just set either of these as the default?

Reply:
I think 99.9% is fine given this is an application that would probably want higher accuracy